fedora-builder/src/main/java/org/dataconservancy/nihms/builder/fs/PassJsonFedoraAdapter.java

```diff
@@ -287,7 +300,17 @@ public Submission fcrepoToPass(URI submissionUri, HashMap<URI, PassEntity> entit
             }
         }
 
-        // TODO - Search for Files resources that reference this Submission and include them in the entity list.
+        // Add File resources that reference this Submission to the entity list.
+        Map<String, Collection<URI>> incomingLinks = client.getIncoming(submissionUri);
+        Collection<URI> uris = incomingLinks.get(Submission.class.getSimpleName().toLowerCase());
+        for (URI uri : uris) {
+            try {
+                File file = client.readResource(uri, File.class);
+                entities.put(uri, file);
+            } catch (Exception e) {
```

Comment:
Can we narrow this catch block to catching the specific exception that will be thrown by Jackson when `readResource` fails?  There could be other reasons that the call to `readResource` fails, and we don't want to swallow those exceptions.  The side affect would be that a submission would be missing files, which would be quite a bad thing.

Reply:
Sure.  I tried to do this filtering without relying on exceptions, but the fedora client doesn't let you read a generic PassEntity, which would have been my preference.  I will test again to find the specific exception and then use that in the catch.